fab/environments.yml

```diff
@@ -143,7 +143,7 @@ production:
       email_queue:
         concurrency: 2
       repeat_record_queue:
-        concurrency: 3
+        concurrency: 20
```

Comment:
that was my impression from my local testing and reading about celery pooling: http://docs.celeryproject.org/en/latest/userguide/concurrency/eventlet.html. i know it's not gevent but i think the same idea applies